ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala

```diff
@@ -68,6 +69,7 @@ object HttpServiceTestFixture extends LazyLogging with Assertions with Inside {
       leakPasswords: LeakPasswords = LeakPasswords.FiresheepStyle,
       useTls: UseTls = UseTls.NoTls,
       wsConfig: Option[WebsocketConfig] = None,
+      nonRepudiation: Option[(Path, Path, String)] = None,
```

Comment:
This triple deserves a type. If you want to factor it with the other triple where the values are `Option` instead of `Id`, that can be a single higher-kinded type; if not, it deserves two types, one with `Option`, one without (where `validateNonRepudiationConfiguration` converts one to the other format).

Reply:
Very good point, addressed in https://github.com/digital-asset/daml/pull/9180/commits/57601845e944aa8054d49fc8c91feabfe947a73c.